api/filter/network/tcp_proxy.proto

```diff
@@ -13,15 +13,13 @@ import "validate/validate.proto";
 
 // [#protodoc-title: TCP Proxy]
 // TCP Proxy :ref:`configuration overview <config_network_filters_tcp_proxy>`.
-// [#v2-api-diff: The route match now takes place in the :ref:`FilterChainMatch
-// <envoy_api_msg_FilterChainMatch>` table].
 
 message TcpProxy {
   // The prefix to use when emitting :ref:`statistics
   // <config_network_filters_tcp_proxy_stats>`.
   string stat_prefix = 1 [(validate.rules).string.min_bytes = 1];
 
-  // The upstream cluster to connect to.
+  // [#not-implemented-hide:] The upstream cluster to connect to.
```

Comment:
I don't understand why we're hiding this.  This is implemented, and working correctly AFAICT